### ISSUE TITLE
revive:disable-next-line isn't working

### ISSUE THREAD

**Issue by kevincantu** [mgechev/revive#540]
Using macOS and golangci-lint v1.41.1 (a207480, 2021-06-19), I expected this to work, and not give me a linter failure:
```go
//revive:disable-next-line:var-naming
func (source Source) BaseApiURL() string {
```

But actually I get this error when linting:
```
source.go:99:22: var-naming: method BaseApiURL should be BaseAPIURL (revive)
func (source Source) BaseApiURL() string {
                     ^
make: *** [lint] Error 1
```

To work around, I can use this:
```go
func (source Source) BaseApiURL() string { //revive:disable-line:var-naming
```

But from how `disable-next-line` is described in the README, I think I've seen a bug.

**Comment by chavacava** [comment]
Hi @kevincantu, thanks for filling the bug description. I'll check it ASAP

**Comment by chavacava** [comment]
Hi @kevincantu, can you please check if the PR #543 fix the problem?

I've detected the following bug: when the disable annotation was not the first line in the comment block, the annotation was ignored. Example:

```go
// BaseApiURL yields ...
//revive:disable-next-line:var-naming   <- this annotation was ignored
func (source Source) BaseApiURL() string {}
```

The example you provided in the issue is something that should work without this fix, i.e. I was not able to reproduce the bug with the exact code example you provided.
